text/0021-versioning-releases.md

```diff
@@ -0,0 +1,26 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Decision
+Openmrs spa applications should have releases that are versioned so that each implementor can have control on when to upgrade to the latest changes. 
+
+The releases should have a maintenance branch for bug fixes and patches so that implementors running a particular version of an app can benefit from recent bug fixes. 
+
+Releases should be tagged, and the artifacts distributed via a CDN maintained by OpenMRS, as is currently the case with the openmrs single-spa test environment hosted in digital ocean spaces, so that implementors get only stable resources to choose from. 
+
+We should adapt semantic versioning, as it is the most widely accepted versioning scheme for releases. See https://semver.org/ 
```

Comment:
We should probably have a stronger reason for landing on SemVer than just "it is the most widely accepted versioning scheme". Among other things, this decision is both central to the point of this RFC and a break with how the OpenMRS community currently manages versions, though arguably OpenMRS as a whole should move towards adopting SemVer.

I would favour something like: "as SemVer allows implementors to more easily assess the work involved in upgrading an application".

Reply:
@ibacher OpenMRS has standardized on SemVer I believe, it just might not be fully enforced or applied retroactively to everything

Reply:
@mseaton Fair. I still think the RFC can provide a stronger rationale for this decision.